src/context/WalletContext.ts

```diff
@@ -33,7 +33,8 @@ const WalletContext = /* #__PURE__ */ createContext<TWallet>({
   disconnect: () => {},
   estimateGas: () => Promise.reject(),
   waitForTransaction: () => Promise.reject(),
-  getTransaction: () => Promise.reject()
+  getTransaction: () => Promise.reject(),
+  evmSendTokens: () => Promise.resolve()
```

Comment:
- `evm` -> `ERC20` (more accurate name)
- `tokens` -> `token` (not array)
- action: start with a verb
```suggestion
  sendERC20Token: () => Promise.resolve()
```